uwh-refbox/src/style.css

```diff
@@ -267,6 +268,10 @@ button.gray:disabled {
 
 
 
+label.*:disabled {
+    background-color: @disabled_gray;
+    color: @text_disabled_gray;
+}
```

Comment:
The `.*:` syntax doesn't seem to work here, I think we need to fix as discussed:
```
label:disabled,
label.edit-parameter-header:disabled {
```

Reply:
ok